newsroom/mgmt_api/utils.py

```diff
@@ -0,0 +1,36 @@
+import bson
```

Comment:
Looks like this file is not used yet, would be good to keep this initial PR minimal and add this code later in another task